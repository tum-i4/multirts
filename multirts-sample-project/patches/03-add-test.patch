diff --git a/multirts-sample-project/bar/src/test/java/a/b/c/bar/FooBarTest.java b/multirts-sample-project/bar/src/test/java/a/b/c/bar/FooBarTest.java
new file mode 100644
index 0000000..7c47019
--- /dev/null
+++ b/multirts-sample-project/bar/src/test/java/a/b/c/bar/FooBarTest.java
@@ -0,0 +1,13 @@
+package a.b.c.bar;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+class FooBarTest {
+
+    @Test
+    void shouldBarReturnFalse() {
+        assertFalse(Bar.bar());
+    }
+}
