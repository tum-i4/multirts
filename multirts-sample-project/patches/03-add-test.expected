module-selection:
    modules:
        bar/pom.xml

test-selection:
    tests:
        a.b.c.bar.FooBarTest

    modules:
        bar/pom.xml